packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -171,34 +172,42 @@ export class EthereumAnchorValidator implements AnchorValidator {
       // determine network based on a chain ID
 
       const provider: providers.BaseProvider = this._getEthProvider(chainId)
-      const transaction: TransactionResponse = await this._getTransaction(provider, txHash)
-
-      if (!transaction) {
-        if (!this.ethereumRpcEndpoint) {
-          throw new Error(
-            `Failed to load transaction data for transaction ${txHash}. Try providing an ethereum rpc endpoint`
-          )
-        } else {
-          throw new Error(`Failed to load transaction data for transaction ${txHash}`)
+      const transaction: TransactionResponse = await backOff(async () => {
+        const tx = await this._getTransaction(provider, txHash)
+        if (!tx) {
+          if (!this.ethereumRpcEndpoint) {
```

Comment:
@stbrody Due to lack of familiarity, can this endpoint change, or should this check be pulled out of the backoff code?

Reply:
nope, can't change in a running instance, requires a restart to update